resources/measures/HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1108,33 +1108,30 @@ def self.process_ducts_for_unit(model, runner, ducts, building, unit, unit_index
       ah_return_leakage = ducts.ah_return_frac * ducts.total_leakage
     end
 
-    # Fraction of ducts in primary duct location (remaining ducts are in above-grade conditioned space).
-    location_frac_leakage = Airflow.get_location_frac_leakage(ducts.location_frac, num_stories)
-
-    location_frac_conduction = location_frac_leakage
+    # Calculate duct areas
+    # Based on ASHRAE Standard 152 (https://www.energy.gov/eere/buildings/downloads/ashrae-standard-152-spreadsheet)
+    f_out = Airflow.get_location_frac_leakage(ducts.location_frac, num_stories) # Fraction of ducts in primary duct location (remaining ducts are in above-grade conditioned space).
```

Comment:
For consistency with ASHRAE 152, you should be passing in `building.stories`, not `num_stories`, as the logic is based on the [number of conditioned floors above-grade](https://github.com/NREL/OpenStudio-HPXML/blob/ffbc9fdae4cb16fac8b9bddabb787ef883ecceee/HPXMLtoOpenStudio/resources/hvac.rb#L4019).